server/registry/internal/storage/models/api.go

```diff
@@ -104,7 +104,7 @@ func (api *Api) Message() (message *rpc.Api, err error) {
 	return message, nil
 }
 
-// Update modifies a api using the contents of a message.
```

Comment:
These probably go back to early times when "api" was called "product" and the article wasn't fixed in the renaming.